test-suite/shared/src/test/scala/org/scalajs/testsuite/javalib/util/MapTest.scala

```diff
@@ -29,7 +31,14 @@ trait MapTest {
 
   def factory: MapFactory
 
+  def testObj(i: Int): TestObj = TestObj(i)
+
+  protected def assumeFalseIdentityHashMap(msg: String = "JVM vs JS cache differences"): Unit =
```

Comment:
```suggestion
  protected def assumeNotIdentityHashMapOnJVM(msg: String = "JVM vs JS cache differences"): Unit =
```
Also the `msg` is never overridden. Consider removing the parameter altogether, and inlining the message string in the call to `assumeFalse`.

Reply:
Done. It was overridden in `IdentityHashMapTest` but removed by your other suggestion to only omit the tests that fail on JVM.